packages/forma-36-website/src/pages/components/icon/index.mdx

```diff
@@ -0,0 +1,207 @@
+---
+title: 'Icon'
+type: 'component'
+github: 'https://github.com/contentful/forma-36/tree/master/packages/forma-36-react-components/src/components/Icon'
+storybook: 'https://f36-storybook.contentful.com/?path=/story/components-icon--icon-default'
+---
+
+(Icons are visual hints for users to indicate actions, statuses and so on. Is used in Buttons, Icon Buttons and so on.)
```

Comment:
Icons can make content more scannable by providing a visual representation of a subject. Icons are used to indicate actions, statuses and navigation. The Icon component is used in Buttons, Icon Buttons, Notes and more.